ReactCommon/jsc/JSCRuntime.cpp

```diff
@@ -396,6 +396,13 @@ JSCRuntime::JSCRuntime(JSGlobalContextRef ctx)
       stringCounter_(0)
 #endif
 {
+// [macOS
+#ifndef NDEBUG
```

Comment:
ifndef is evil because even setting to =0 means it defines it and so dumb people like me get confused why things are compiling and then duh moment.